icu4c/source/Makefile.in

```diff
@@ -169,7 +171,11 @@ all-local: $(srcdir)/configure $(LOCAL_BUILT_FILES) $(INSTALLED_BUILT_FILES)
 ifndef VERBOSE
 	@echo "Note: rebuild with \"$(MAKE) VERBOSE=1 $(MAKECMDGOALS)\" to show all compiler parameters."
 endif
+ifeq ($(INSTALL_ICU_CONFIG),true)
 install-local: install-icu install-manx
```

Comment:
Minor, I see what this is trying to do, but could it instead  conditionalize the line 69:

```
ifeq  ($(INSTALL_ICU_CONFIG),true)
MANX_FILES += config/icu-config.$(SECTION)
endif
```

then manx-files becomes a no-op but the mechanism is still there if other man files show up (it could happen…)

Reply:
Fine by me.

Reply:
Actually, this doesn't work. `make install` fails because `/usr/bin/install` expects a file as an argument. We can solve this by checking whether `MANX_FILES` is set.

    ifneq ($(MANX_FILES),)
            $(INSTALL_DATA) $? $(DESTDIR)$(mandir)/man$(SECTION)
    endif
Is this acceptable?

Reply:
@hughmcmaster sounds good. thanks.